tools/plugins/com.liferay.ide.project.ui/templates/project-java-templates.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<templates>
+
+<template name="component" description="Get property for the current component class" id="org.eclipse.jdt.ui.templates.component" context="java" enabled="true" autoinsert="false">
```

Comment:
So if this template is for a generic `component` then it should not specify the properties for a `portlet` component.  Its property list should be empty.  And then we could create several other templates as well, one for each of the new component templates that are generated by our new Component wizard.